### PR TITLE
Update version and changelog for 6.15.0

### LINKED ISSUE #533: Changelog Entries needed for 6.15.0
This issue tracks those issues or externally submitted PRs that require a CHANGELOG entry.  Attach entries as appropriate, or review [this list of 6.15.0 tagged issues](https://github.com/newrelic/newrelic-ruby-agent/issues?q=label%3Aversion%3A6.15.0) for completeness.

### PR DESCRIPTION
**Pull request by tannalynn**
_Before contributing, please read our [contributing guidelines](https://github.com/newrelic/newrelic-ruby-agent/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/newrelic/.github/blob/master/CODE_OF_CONDUCT.md)._

# Overview
Bumps the version and updates the changelog

# Related Github Issue
closes #533

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
   ## v6.15.0
 
+  * **Official Ruby 3.0 support**
+    
+    The ruby agent has been verified to run on ruby 3.0.0
+
+  * **Added support for Rails 6.1**
+
+    The ruby agent has been verified to run with Rails 6.1
+
```
Comment (mwlang) [inline CHANGELOG.md:12]: Please call out our contributor with special thanks here!

**Review by mwlang — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tannalynn) [reply CHANGELOG.md]: Great point! thank you for the reminder!

**Review by mwlang — APPROVED** [review]
Looks good.

**Comment by olleolleolle** [comment]
Sorry to snoop around – this didn't yet make a release to RubyGems, is that a separate step?

**Comment by angelatan2** [comment]
Hello @olleolleolle, We are in the process of releasing.  Yes, please treat what you see in the changelog for the moment as staging.

**Pull request merged**